We don't have pytest-runner in Solaris, and since this package adds
nothing we need, we can safely remove this setup dependency.

Not suitable for upstream.

--- pylint-2.3.1/setup.py
+++ pylint-2.3.1/setup.py
@@ -169,7 +169,7 @@ def install(**kwargs):
                  extras_require=extras_require,
                  test_suite='test',
                  python_requires='>=3.4.*',
-                 setup_requires=['pytest-runner'],
+                 setup_requires=[],
                  tests_require=['pytest'],
                  **kwargs)
 
